src/utilities/permissions.js

```diff
@@ -0,0 +1,182 @@
+const permissions = {
+  Administrator: [
+    'seeBadges',
+    'assignBadges',
+    'createBadges',
+    'deleteBadges',
+    'updateBadges',
+    'createPopup',
+    'updatePopup',
+    'deleteProject',
+    'postProject',
+    'putProject',
+    'assignProjectToUsers',
+    'importTask',
+    'postTask',
+    'updateNum',
+    'updateTask',
+    'swapTask',
+    'deleteTeam',
+    'putTeam',
+    'assignTeamToUsers',
+    'editTimeEntry',
+    'deleteTimeEntry',
+    'updatePassword',
+    'getUserProfiles',
+    'getProjectMembers',
+    'postUserProfile',
+    'putUserProfile',
+    'putUserProfileImportantInfo',
+    'infringmentAuthorizer',
+    'deleteUserProfile',
+    'postWbs',
+    'deleteWbs',
+    'getAllInvInProjectWBS',
+    'postInvInProjectWBS',
+    'getAllInvInProject',
+    'postInvInProject',
+    'transferInvById',
+    'delInvById',
+    'unWasteInvById',
+    'getInvIdInfo',
+    'putInvById',
+    'getInvTypeById',
+    'putInvType',
+    'getAllInvType',
+    'postInvType',
+    'getWeeklySummaries',
+    'getTimeZoneAPIKey',
+    'checkLeadTeamOfXplus',
+  ],
```

Comment:
Based on the user class document, `Core Team` seem is missing here.
<img width="362" alt="image" src="https://user-images.githubusercontent.com/5071040/167966540-932a8f6c-64a2-45c8-9f76-045a66a94056.png">